libkbfs/block_retrieval_queue_test.go

```diff
@@ -71,9 +71,15 @@ func makeKMD() KeyMetadata {
 	return emptyKeyMetadata{tlf.FakeID(0, tlf.Private), 1}
 }
 
+func initBlockRetrievalQueueTest(t *testing.T) *blockRetrievalQueue {
+	q := newBlockRetrievalQueue(0, 0, newTestBlockRetrievalConfig(t, nil, nil))
+	<-q.TogglePrefetcher(false, nil)
```

Comment:
It feels like the test should be waiting for the prefetcher to shut down, but I don't have a great reason why other than that it might be confusing for debugging if there are still goroutines from old prefetchers lying.  But I guess since `TogglePrefetcher(false)` doesn't actually set the prefetcher to nil, the queue shutdown method will still end up waiting for the shutdown to finish.  If that sounds right, nevermind me.

Reply:
The channel returned by `TogglePrefetcher` is the channel from the previous prefetcher's `Shutdown`, so that's exactly what we're doing here.

Reply:
D'oh!  I didn't notice the `<-` somehow.  Nevermind.